tools/interactive/interactivetool_geoexplorer.xml

```diff
@@ -20,6 +20,8 @@
 
         mkdir -p /srv/shiny-server/data/ &&
         cp '$infile' /srv/shiny-server/data/inputdata.txt &&
+        chmod 777 /srv/shiny-server/data/inputdata.txt &&
```

Comment:
you should change that for a proper version in the contianer

Reply:
Ok! So you mean, changing something on the R Shiny app code? Because on the Dockerfile, I don't know how I can prepare this as it seems there is an issue for the R shiny app to access a file copied by Galaxy ...

Reply:
Oh.... Wait.... I am just thinking that maybe this is just on my local Galaxy instance I have to chmod 777 the files, and maybe on usegalaxy.eu, this is ok. And yes! In fact, This is due to a Galaxy instance config I think